libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -267,6 +267,28 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateLinksStrip = (content) => {
```

Comment:
Should this be only loaded when actually used? Same for the CSS. @overmyheadandbody knows more, but I think this is only used on a small sub-section of pages

Reply:
In theory, this should be used in a wide range of places, except the homepage, where we already have all the clouds; and BACOM, where they don't seem to be showing the cloud menu.
* https://www.adobe.com/products/photoshop.html - CC subpages have it
* https://www.adobe.com/acrobat.html - DC subpages have it

However, this won't be authored in the near future, I'd assume, so it might be dead code for a while. I wouldn't necessarily do a dynamic import, as it's already part of `menu.js`, which is loaded later in the flow anyway. I'd avoid longer chains of dynamic imports and instead focus on failing fast and having a low code footprint for the new functionality.

Reply:
Sounds reasonable! No action required in this case